src/System.Runtime/samples/DateTime-formatting/DateTime-formatting.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Globalization;
+
+public class Program
+{
+	public static void Main()
+	{
+		// Demonstrates common uses of formatting the DateTime struct.
+		// Doc: https://msdn.microsoft.com/library/az4se3k1.aspx
+		
+		var date = DateTime.Now;
+		
+		// default formatting
+		Console.WriteLine("Print DateTime with default formatting: {0}", date);
+		Console.WriteLine();
+		
+		// short / long formatting
+		Console.WriteLine("Print short date: {0}", date.ToString("d"));
+		Console.WriteLine("Print long date: {0}", date.ToString("D"));
+		Console.WriteLine("Print short time: {0}", date.ToString("t"));
+		Console.WriteLine("Print long time: {0}", date.ToString("T"));
+		Console.WriteLine();
+		
+		// custom formatting with templating
+		Console.WriteLine("Print date with month/year format: {0}", date.ToString("MM/yy"));
+		Console.WriteLine("Print date with month, year in verbose format: {0}", date.ToString("MMMM, yyyy"));
+		Console.WriteLine("Print time: {0}", date.ToString("hh:mm:ss"));
+		Console.WriteLine();
+		
+		// print with locale-specific formatting
+		var ptBRCulture = new CultureInfo("pt-BR");
```

Comment:
Similarly, these should be CultureInfo.GetCultureInfo(...) instead of using the ctor. GetCultureInfo returns cached culture.